crit/Makefile

```diff
@@ -22,6 +22,10 @@ update-proto:
 	# from other projects, the files of this repository are
 	# now registered under the package name `criu`.
 	ls ./images/*.proto | xargs sed -i '/syntax/a package criu;'
```

Comment:
I think we should revert this change, since it introduces another layer of inconsistency with the original images from the criu repo. If our only solution is to rename the files to `criu-sa.proto` and `criu-core.proto`, then the package directive does not seem to benefit us in any way. We can either revert the changes in this PR, or revert the commit introduced in #90 before releasing v6.2.0. What do you think?

Reply:
Yes, if the package `criu` does not gain us anything it is not necessary and just increases the difference between `go-criu` and `criu`.

Related to that, a bit, I am always a bit unhappy when I see that importing `go-criu` requires that all `*proto` files are imported even if unneeded. Is there a way to have each file in its own directory. That probably breaks `import` all over the place. As we are renaming files and changing `import` statements already... But that would be to big a change.

The thing I don't like about this change is that we need to rename files every time  a new collision happens and so far none of the projects importing `go-criu` currently need `sa.proto` and `core.proto`. It would be good to have an abstraction in Go which does only import the files needed and not everything. So far none of the tools using `go-criu` includes the phaul directory as it is not imported in the Go code. Can we make a change to image files in such a way that only the needed files are imported and not everything. Because that leads to the problems we currently have.

Reply:
> It would be good to have an abstraction in Go which does only import the files needed and not everything

@adrianreber Unfortunately, Go imports all files that may _potentially_ be involved in the package. Since `crit/images/handler.go` has all the image types in the switch cases, Go will include all of the `.pb.go` files, and by extension all of the `.proto` files. I too dislike this form of package management, but I do not think there's any workaround we can do from our side :(

Reply:
At this point it feels we tried everything and nothing really helps us.

Creating a subdirectory for each `*proto` file also does not work as we have `import` in the `*proto` files we would need to be adapted.

The best solution would be that CRIU renames all files to `criu-*`. But this also sounds difficult to get merged. Maybe it is worth a try at some point. Anyway... Sounds like we have to merge it the way it is now.